internal/telemetry/telemetry.go

```diff
@@ -92,7 +92,7 @@ func (r *Reporter) report(_ context.Context, info info.Flipt, f file) error {
 		r.logger.Debug("initialized new state")
 	} else {
 		t, _ := time.Parse(time.RFC3339, s.LastTimestamp)
-		r.logger.Debugf("last report was: %v ago", time.Since(t))
+		r.logger.Debug("last report", zap.Time("when", t), zap.Duration("ellapsed", time.Since(t)))
```

Comment:
```suggestion
		r.logger.Debug("last report", zap.Time("when", t), zap.Duration("elapsed", time.Since(t)))
```

Reply:
I've done this so many times. This typo 😂